src/Index.js

```diff
@@ -46,7 +46,7 @@ Index.prototype.addObject = function(content, objectID, callback) {
     hostType: 'write',
     callback: callback
   });
-};
+}, deprecatedMessage('index.addObject()', 'index.addObjects()'));
```

Comment:
It's a tough one, I would say we should accept only the array for consistency and precision but still have good errors in term of usage.

variadic, boolean traps, magic obj => [obj] decisions are most of the time bad choices we regret later on.